src/logger.cc

```diff
@@ -133,12 +134,18 @@ NAN_METHOD(Logger::New) {
           const std::string fileName = *Nan::Utf8String(info[2]);
 #endif
 
-          logger = spdlog::rotating_logger_mt(
+          if (logName == "rotating_async") {
+            logger = spdlog::rotating_logger_st<spdlog::async_factory>(
+              logName, fileName, Nan::To<int64_t>(info[3]).FromJust(),
```

Comment:
Define `SPDLOG_WCHAR_FILENAMES` in binding.gyp for windows to fix compilation error.